orders/models.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+from django.utils import timezone
+
+
+class StatusOptions(models.TextChoices):
+    Pending = 'P', 'Pending'
+    Approved = 'A', 'Approved'
+    Canceled = 'C', 'Canceled'
+    Finished = 'F', 'Finished'
+    OnGoing = 'O', 'On Going'
+
+
+class Order(models.Model):
+    order_id = models.AutoField(primary_key=True)
+    dog_owner_id = models.IntegerField()
+    daycare_id = models.IntegerField()
+    booking_date = models.DateTimeField(auto_now=True)
```

Comment:
I think that the fields: `dog_owner_id` and `daycare_id` should be mentioned as FK

Reply:
> I think that the fields: `dog_owner_id` and `daycare_id` should be mentioned as FK


They weren't FK until the merge of DogOwner model.
Changed it afterwards.